go/vt/vttablet/tabletserver/vstreamer/vstreamer_test.go

```diff
@@ -118,18 +118,28 @@ func TestSetAndEnum(t *testing.T) {
 	ts := &TestSpec{
```

Comment:
Are we testing for null values in enum/set columnss (for nullable columns)?  Or empty enums/sets in non-nullable?

Reply:
Good point. I'll add some.